monkey/common/cloud/azure/azure_instance.py

```diff
@@ -30,30 +31,29 @@ def __init__(self):
         self.instance_name = None
         self.instance_id = None
         self.location = None
-        self.on_azure = False
+        self._on_azure = False
 
         try:
             response = requests.get(AZURE_METADATA_SERVICE_URL,
                                     headers={"Metadata": "true"},
                                     timeout=SHORT_REQUEST_TIMEOUT)
-            self.on_azure = True
 
             # If not on cloud, the metadata URL is non-routable and the connection will fail.
             # If on AWS, should get 404 since the metadata service URL is different, so bool(response) will be false.
             if response:
-                logger.debug("On Azure. Trying to parse metadata.")
+                logger.debug("Trying to parse Azure metadata.")
                 self.try_parse_response(response)
             else:
-                logger.warning("On Azure, but metadata response not ok: {}".format(response.status_code))
+                logger.warning(f"On Azure, but metadata response not ok: {response.status_code}")
```

Comment:
This log message is inaccurate

```suggestion
                logger.warning(f"Metadata response not ok: {response.status_code}")
```